recipe/meta.yaml

```diff
@@ -163,7 +163,11 @@ requirements:
     # http://doc.qt.io/Qt-5/qdoc-guide-clang.html
     # Only libclang is needed, but needs the full package for detection.
     - llvmdev
-    - clangdev
+    # Ensure that build and host have the same clang version on osx
+    # otherwise qt6 may be out of sync compared to the pinned compiler
+    # version
+    - clangdev {{ cxx_compiler_version }}  # [osx]
+    - clangdev                             # [not osx]
```

Comment:
If you need libclang, why not specify that?

The libclang ABI has been stable from 13-15 (output name `libclang13`), so strictly speaking there wouldn't even be a need to rebuild.

Reply:
Ok, i moved the pinning as you suggested.

Reply:
Truthfully, I'm not too sure about these pinnings. However, we should keep these libraries in sync if anything.